iron-list.html

```diff
@@ -657,12 +721,16 @@
     _ensureTemplatized: function() {
       if (!this.ctor) {
         // Template instance props that should be excluded from forwarding
-        this._instanceProps = {
-          __key__: true
-        };
-        this._instanceProps[this.as] = true;
-        this._instanceProps[this.indexAs] = true;
+        var props = {};
+
+        props.__key__ = true;
+        props[this.as] = true;
+        props[this.indexAs] = true;
+        props[this.selectedAs] = false;

```

Comment:
I think templatizer only knows about the keys of `_instanceProps`, but it seems strange to be `false` :)